ci/accelerator.sh

```diff
@@ -31,22 +31,23 @@ progs=(
     "streaming_accelerator_loopback"
     "streaming_accelerator_vdp"
     "coherent_accelerator_vdp"
-    "streaming_accelerator_loopback"
-    "coherent_accelerator_vdp"
     )
 
+#    "streaming_accelerator_loopback"
```

Comment:
I believe this doesn't test the 4x configs?

Reply:
I disabled after realizing the confusing accelerator config schemes. I think I fixed the configs now, so I will re-enable and test.

Reply:
tests seem to pass, re-enabled.